docs/charts.md

```diff
@@ -25,6 +25,7 @@ wordpress/
   Chart.yaml        # A YAML file containing information about the chart
   LICENSE           # OPTIONAL: A plain text file containing the license for the chart
   README.md         # OPTIONAL: A human-readable README file
+  NOTES.txt         # OPTIONAL: A plain text file containing short usage notes
```

Comment:
The only issue I see with that is that it'll be harder for people to see when browsing a chart repository on GitHub. However, a NOTES.txt that makes good use of templating wouldn't really be human-readable in the raw form anyway. If it makes the rendering easier, I'm okay moving it to `templates/NOTES.txt`.